modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletedItems.testcase

```diff
@@ -339,11 +339,30 @@ definition {
 	}
 
 	@description = "LPS-94146 - Verify that when there are no results for a filter selection, the table should display the respective empty state"
-	@ignore = "Test Stub"
 	@priority = 3
 	test TableDisplaysEmptyStateWhenThereIsNoResult {
 
-		// TODO LRQA-67647 TableDisplaysEmptyStateWhenThereIsNoResult pending implementation
+		task("Given a workflow"){
```

Comment:
Added test from https://liferay.atlassian.net/browse/LRQA-67647 and added task blocks.